.github/workflows/jax-vllm-offloading-eks-transfer.yml

```diff
@@ -0,0 +1,52 @@
+name: JAX-vLLM offloading transfer (EKS)
+
+on:
+  workflow_call:
+    inputs:
+      JAX_VLLM_OFFLOADING_IMAGE:
+        type: string
+        description: MaxText image from ghcr.io/nvidia
+        default: ghcr.io/nvidia/jax-toolbox-internal:19758308493-jio-amd64
```

Comment:
Before we have a nightly image, we can remove the default so that the workflow fails loudly if no image is specified.